src/lib/Logger.ts

```diff
@@ -1,18 +1,29 @@
 import * as winston from 'winston';
+import { isMatch } from 'micromatch';
 
 process.stdout.isTTY = true;
 
 const ProdLogger = winston.createLogger({
-  level: 'info',
+  level: 'http',
```

Comment:
This isnt going to remove all info messages from prod is it?

Reply:
This means that prod logs all levels `http` and below. 
Levels are defined as:
```
{ 
  error: 0, 
  warn: 1, 
  info: 2, 
  http: 3,
  verbose: 4, 
  debug: 5, 
  silly: 6 
}
```